packages/editor/src/store/reducer.js

```diff
@@ -360,6 +360,17 @@ export function listViewToggleRef( state = { current: null } ) {
 	return state;
 }
 
+/**
+ * This reducer does nothing aside initializing a ref to the inserter sidebar toggle.
+ * We will have a unique ref per "editor" instance.
+ *
+ * @param {Object} state
+ * @return {Object} Reference to the inserter sidebar toggle button.
+ */
+export function inserterSidebarToggleRef( state = { current: null } ) {
```

Comment:
Not sure if it matters, but every time this function is called, the state will be a different instance of ref. Probably better this way:

```suggestion
const INITIAL_REF = createRef( null );
export function inserterSidebarToggleRef( state = INITIAL_REF ) {
```